.github/workflows/scrape.yml

```diff
@@ -3,11 +3,9 @@ name: Scrape
 
 on:
   push:
-    branches:
-      - data-auto
-  schedule:
-    - cron: "*/5 * * * *"
-  # Allow running manually
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
```

Comment:
En fait c'est quoi l'idée d'avoir une action scrape séparée sans publish ? Pour tester ? Est-ce que ça vaut vraiment le coup de scrape à chaque PR sur main et à chaque push sur main ? Charge inutile sur les API amha, il vaut mieux investir dans des tests.

Reply:
non, je pense que c'est pas une bonne idée ^^
dans l'idéal il faudrait qu'on puisse lancer scrape manuellement pour sa PR et que ce soit difficile de lancer scrape and publish par erreur

Reply:
allez j'ai viré pour l'instant, on verra plus tard